src/scripts.js

```diff
@@ -1,35 +1,76 @@
+
+// IMPORTS
 import './css/base.scss';
 import './css/styles.scss';
-
-import recipeData from './data/recipes';
-import ingredientsData from './data/ingredients';
-import users from './data/users';
-
 import Pantry from './pantry';
 import Recipe from './recipe';
 import User from './user';
 import Cookbook from './cookbook';
+import UserRepository from './userRepository'
+
+// GLOBALS
+let wcUsersData;
+let userRepo;
+let recipeData;
+let ingredientsData;
+let user, pantry;
+
+// Fetching
+wcUsersData = fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData')
+.then(data => data.json())
+.then(data => data.wcUsersData)
+.catch(err => console.log(err.message))
+
+ingredientsData = fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/ingredients/ingredientsData')
+.then(data => data.json())
+.then(data => data.ingredientsData)
+.catch(err => console.log(err.message))
+
+recipeData = fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/recipes/recipeData')
+.then(data => data.json())
+.then(data => data.recipeData)
+.catch(err => console.log(err.message))
+
+//PROMISE
+Promise.all([wcUsersData, ingredientsData, recipeData])
+.then(data => {
+  wcUsersData = data[0];
+  ingredientsData = data[1]
+  recipeData = data[2]
+})
+.then(() => {
+  userRepo = new UserRepository(wcUsersData);
+
+  onStartup(wcUsersData)
+})
+.catch(error => {
+  console.log('Something is amiss with promise all', error)
+});
 
 let favButton = document.querySelector('.view-favorites');
 let homeButton = document.querySelector('.home')
 let cardArea = document.querySelector('.all-cards');
-let cookbook = new Cookbook(recipeData);
-let user, pantry;
 
-window.onload = onStartup();
+
 
 homeButton.addEventListener('click', cardButtonConditionals);
 favButton.addEventListener('click', viewFavorites);
 cardArea.addEventListener('click', cardButtonConditionals);
 
-function onStartup() {
-  let userId = (Math.floor(Math.random() * 49) + 1)
-  let newUser = users.find(user => {
-    return user.id === Number(userId);
-  });
-  user = new User(userId, newUser.name, newUser.pantry)
-  pantry = new Pantry(newUser.pantry)
+function onStartup(wcUsersData) {
```

Comment:
Nice adjustments to the onStartup method!

Reply:
Dope, let's remember to check back in on their naming conventions, once we're up and running like we want to be.